script/update-doc-repo.pl

```diff
@@ -852,12 +866,13 @@ sub process_minion_asciidoc_docdir {
 		system('rsync', '-rl', '--no-compress', '--delete', $files.'/', $filedir.'/') == 0 or die "Failed to sync $files to $filedir: $!\n";
 	}
 
-	opendir(DIR, $docdir) or die "Failed to open $docdir for reading: $!\n";
-	for my $entry (sort { lc($a) cmp lc($b) } grep { /\.html$/ } readdir(DIR)) {
+	my $DOCDIR_HANDLE;
+	opendir($DOCDIR_HANDLE, $docdir) or die "Failed to open $docdir for reading: $!\n";
+	for my $entry (sort { lc($a) cmp lc($b) } grep { /\.html$/ } readdir($DOCDIR_HANDLE)) {
 		my ($name) = $entry =~ /^(.*?)\.html$/;
 		$name = lc($name);
 
-		my $target = File::Spec->catdir($INSTALLDIR, $name);
+		my $target = File::Spec->catdir($INSTALL$DOCDIR_HANDLE, $name);
```

Comment:
I think this was mistakenly changed.

Reply:
I went ahead and pushed the fix for this in 09221ba03061f827f11745fc76aa389094aefa11.